llvm/utils/TableGen/DAGISelEmitter.cpp

```diff
@@ -165,8 +165,21 @@ void DAGISelEmitter::run(raw_ostream &OS) {
   // Add all the patterns to a temporary list so we can sort them.
   Records.startTimer("Sort patterns");
   std::vector<const PatternToMatch *> Patterns;
-  for (const PatternToMatch &PTM : CGP.ptms())
+  for (const PatternToMatch &PTM : CGP.ptms()) {
```

Comment:
I thought this only affected GlobalISel's import of DAG patterns, why does DAGISelEmitter need changes too?